include/deal.II/multigrid/mg_transfer_global_coarsening.h

```diff
@@ -706,6 +706,32 @@ class MGTwoLevelTransferNonNested<dim,
   using VectorizedArrayType = VectorizedArray<Number, 1>;
 
 public:
+  /**
+   * AdditionalData structure that can be used to tweak parameters
+   * @p tolerance and @p rtree_level related to the search procedure (used
+   * internally by RemotePointEvaluation) or, in the future, transfer operators
+   * needed by the non-nested multigrid algorithm. Default values are the same
+   * as the ones described in the documentation of RemotePointEvaluation.
+   * The last Boolean parameter, @p enforce_all_points_found is true by default
+   * and checks if RemotePointEvaluation::all_points_found() evaluates to true,
+   * i.e. all submitted points have been found inside the domain.
```

Comment:
Normally, we described these parameters closer to the parameters. See https://github.com/dealii/dealii/blob/dfb435053092192fda1ca9a76a1597e589155275/include/deal.II/lac/solver_gmres.h#L197-L261.